zerver/tests/test_signup.py

```diff
@@ -406,8 +406,8 @@ def test_multi_user_invite(self):
         self.login("hamlet@zulip.com")
         # Intentionally use a weird string.
         self.assert_json_success(self.invite(
-"""bob-test@zulip.com,     carol-test@zulip.com,
-dave-test@zulip.com
+            """bob-test@zulip.com,     carol-test@zulip.com,
+            dave-test@zulip.com
```

Comment:
I believe that the endpoint is stripping the whitespace. So while the test is technically different, it's just a different weird string with extra whitespace (the test is already testing weird whitespace). I think it's ok.